### ISSUE TITLE
Add try except around RID lookup

### ISSUE THREAD

**Issue by samaloney** [i4Ds/STIXCore#397]
If the RID isn't found in the table a key error is raised preventing the fits files being created.

Some examples
```
solo_LB_stix-21-6-21_0000000000-9999999999_V02_2404213657-64616.fits - KeyError: 'No matches found for key 2404213657' # CPD
solo_LB_stix-21-6-21_0000000000-9999999999_V02_2405017696-64617.fits - KeyError: 'No matches found for key 2405017696' # CPD
solo_LB_stix-21-6-21_0000000000-9999999999_V02_2404206759-64619.fits - KeyError: 'No matches found for key 2404206759' # CPD
solo_LB_stix-21-6-21_0000000000-9999999999_V02_2405066983-64618.fits - KeyError: 'No matches found for key 2405066983' # CPD
solo_LB_stix-21-6-21_0000000000-9999999999_V02_2404189403-64620.fits - KeyError: 'No matches found for key 2405066983' # CPD
solo_LB_stix-21-6-21_0000000000-9999999999_V02_2404132170-64621.fits - KeyError: 'No matches found for key 2404132170' # CPD
solo_LB_stix-21-6-24_0000000000-9999999999_V02_2406111879-52942.fits - KeyError: 'No matches found for key 2406111879' # SPEC
```

A different error just occurred in a recent TM processing requests:
https://pub099.cs.technik.fhnw.ch/fits/logging/TM_hourly_req_BatchRequest.PktTmRaw.SOL.0.2024.184.14.22.34.499.bBMJ@2024.267.19.00.02.179.1.xml.log.err
https://pub099.cs.technik.fhnw.ch/fits/logging/TM_hourly_req_BatchRequest.PktTmRaw.SOL.0.2024.184.14.22.34.499.bBMJ@2024.267.19.00.02.179.1.xml.log.err

```
/data/stix/SOLSOC/from_edds/tm/processing/TM_hourly_req_BatchRequest.PktTmRaw.SOL.0.2024.184.14.22.34.499.bBMJ@2024.267.19.00.02.179.1.xml
2024-09-23T21:30:12Z INFO __main__ 518: Modified directory: /data/stix/SOLSOC/from_edds/tm/processing
2024-09-23T21:30:25Z ERROR stixcore.io.RidLutManager 168: RID API ERROR
Traceback (most recent call last):
  File "/home/stixcore/STIXCore/venv/lib/python3.9/site-packages/stixcore/io/RidLutManager.py", line 157, in read_rid_lut
    urllib.request.urlretrieve(update_url, updatefile)
  File "/usr/lib/python3.9/urllib/request.py", line 239, in urlretrieve
    with contextlib.closing(urlopen(url, data)) as fp:
  File "/usr/lib/python3.9/urllib/request.py", line 214, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.9/urllib/request.py", line 523, in open
    response = meth(req, response)
  File "/usr/lib/python3.9/urllib/request.py", line 632, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.9/urllib/request.py", line 561, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.9/urllib/request.py", line 494, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.9/urllib/request.py", line 641, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 502: Bad Gateway
2024-09-23T21:30:26Z INFO stixcore.io.RidLutManager 172: write total 33320 entries to local storage
2024-09-23T21:30:26Z INFO stixcore.processing.TMTCtoLB 30: Processing file: SOCPacketFile('/data/stix/SOLSOC/from_edds/tm/processing/TM_hourly_req_BatchRequest.PktTmRaw.SOL.0.2024.234.08.00.27.687.OdhD@2024.267.19.00.03.243.1.xml')
```

**Comment by samaloney** [comment]
If it's the scaling factor we are stuck but if it's only the reason should we default back to an empty string?

**Comment by nicHoch** [comment]
fixed in #411
